homeassistant/components/enocean/cover.py

```diff
@@ -0,0 +1,267 @@
+"""Support for EnOcean roller shutters (EEP D2-05-00)."""
```

Comment:
If there are more devices we won't list them all in the docstring here.
```suggestion
"""Support for EnOcean roller shutters."""
```